docs/snippets/common/chromatic-github-action.js.mdx

```diff
@@ -15,7 +15,13 @@ jobs:
     # Job steps
     steps:
       - uses: actions/checkout@v3
-      - run: yarn
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - run: yarn ci
```

Comment:
@natehouk can you remove the `ci` option from the `yarn` command, as there's no option for it, and as this is a CI workflow, it will automatically set up the [`immutable`](https://yarnpkg.com/cli/install) that operates similarly to the `npm` counterpart.

Reply:
Done

Reply:
@natehouk Sorry for not being explicit. What I meant to say was to remove only the `ci` option, as you'll need to run the `yarn` command to install the dependencies.

Reply:
@jonniebigodes Fixed.

I added back this line:

```
- run: yarn
```

Let me know if this is good now. Thanks.